### PR TITLE
Add -continueOnError flag

### PR DESCRIPTION
**Pull request by andremidea**
The use case for this flag is the following:

We have a daily job that loads automatically ~500 tables, everything is generated programmatically (we generate DDL based on the Spark's Dataset Schema) and sometimes one of the loads fails, given that, all subsequent steps are skipped, this is not great for us because we have some next steps that must be run.

So the idea here is to add the possibility of continuing the execution of the subsequent steps but still return an error code at the end (and also tell which queries failed) 

ps: I signed the CLA

### CODE REVIEW

**Comment by snowplowcla** [comment]
Confirmed! @andremidea has signed the Individual Contributor License Agreement

**Comment by pepijn** [comment]
Can we do anything to have this PR merged? :-)

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.5.2
\ No newline at end of file
+0.6.0
```
Comment (BenFradet) [inline VERSION]: we will take care of that ourselves :+1:

File: sql_runner/main.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ import (
 const (
 	CLI_NAME        = "sql-runner"
 	CLI_DESCRIPTION = `Run playbooks of SQL scripts in series and parallel on Redshift and Postgres`
-	CLI_VERSION     = "0.5.2"
+	CLI_VERSION     = "0.6.0"
```
Comment (BenFradet) [inline sql_runner/main.go]: same

File: sql_runner/options.go [hunk 3]
```diff
@@ -65,6 +66,7 @@ func (o *Options) GetFlagSet() *flag.FlagSet {
 	fs.Var(&(o.variables), "var", "Variables to be passed to the playbook, in the key=value format")
 	fs.StringVar(&(o.fromStep), "fromStep", "", "Starts from a given step defined in your playbook")
 	fs.BoolVar(&(o.dryRun), "dryRun", false, "Runs through a playbook without executing any of the SQL")
+	fs.BoolVar(&(o.continueOnError), "continueOnError", false, "Continues execution even a step fails, the exit code will still be different than 0")
```
Comment (BenFradet) [inline sql_runner/options.go]: `even if a step fails`

**Pull request closed without merging**